client/src/components/UserSignUpForm.js

```diff
@@ -73,48 +97,71 @@ export class UserSignUpForm extends Component {
           ) : (
             <Form onSubmit={this.handleSubmit}>
               <Form.Field>
-                <label>Name</label>
-                <input
+                <Form.Input
+                  icon="user"
+                  iconPosition="left"
                   onChange={this.handleChange}
-                  value={this.state.name}
+                  value={name}
                   placeholder="Name"
                   name="name"
+                  label="Name"
                   required
                 />
-                <label>Email</label>
-                <input
+
+                <Form.Input
+                  icon="envelope"
+                  iconPosition="left"
                   onChange={this.handleChange}
-                  value={this.state.email}
+                  value={email}
+                  label="Email"
                   placeholder="Email"
                   name="email"
                   required
                 />
               </Form.Field>
               <Form.Field>
-                <label>Password</label>
-                <input
+                <Form.Input
+                  type="password"
+                  icon="lock"
+                  iconPosition="left"
                   onChange={this.handleChange}
                   placeholder="Password"
+                  label="Password"
                   name="password"
-                  value={this.state.password}
+                  value={password}
                   required
                 />
               </Form.Field>
               <Form.Field>
-                <label> Password Confirmation</label>
-                <input
-                  value={this.state.confirmationPassword}
+                <Form.Input
+                  type="password"
+                  icon="lock"
+                  iconPosition="left"
+                  label="Password Confirmation"
+                  value={confirmationPassword}
                   onChange={this.handleChange}
                   placeholder="Password confirmation"
                   name="confirmationPassword"
                   required
                 />
               </Form.Field>
               <Form.Field />
-
+              {this.state.hasErrors ? (
+                <div>
+                  <Message negative>
+                    <Message.Header>
+                      We're sorry, we can't create your account
```

Comment:
According to the ticket, the title of the error message box should be: "An error occured"